### PR TITLE
update memberlist to handle overflow and add copying of email

### LINKED ISSUE #565: Visual artefacts when orga member name is to lang
### Describe the bug

as can be seen in the screenshot there is a visual effect when the member of an orga is too long.

### Describe how to produce the bug

edit an orga

### Additional context

grafik

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #565

### CODE REVIEW

File: tasks/components/OrganizationMemberList.tsx [hunk 1]
```diff
@@ -159,9 +159,14 @@ export const OrganizationMemberList = ({
...
           <div key="member" className={tw('flex flex-row items-center h-12')}>
             <Avatar avatarUrl={dataObject.avatarURL} alt="" size="small"/>
-            <div className={tw('flex flex-col ml-2')}>
-              <Span className={tw('font-bold h-5')}>{dataObject.name}</Span>
-              <Span type="description" className={tw('text-sm')}>{dataObject.email}</Span>
+            <div className={tw('flex flex-col ml-2 max-w-[250px] overflow-hidden')}>
+              <Span className={tw('font-bold truncate')}>{dataObject.name}</Span>
+              <button
+                // TODO update to a copy to clipboard button
+                onClick={() => navigator.clipboard.writeText(dataObject.email)}
```
Comment (MaxSchaefer) [inline tasks/components/OrganizationMemberList.tsx:166]: Why did you add this functionality? Mailto link?

Comment (DasProffi) [reply tasks/components/OrganizationMemberList.tsx:166]: mail to link sounds better, but we needed something that allows the user to get the full email, when it is cut off on an overflow

Comment (MaxSchaefer) [reply tasks/components/OrganizationMemberList.tsx:166]: [This is what I meant with "Mailto link".](https://en.wikipedia.org/wiki/Mailto) Maybe we can expand the mail or using a popover. The onClick seams a little bit hidden.

**Review by MaxSchaefer — DISMISSED** [review]

**Review by MaxSchaefer — APPROVED** [review]
